sys/include/net/gnrc/rpl.h

```diff
@@ -137,8 +138,8 @@ extern "C" {
 /**
  * @brief   Default message queue size to use for the RPL thread.
  */
-#ifndef GNRC_RPL_MSG_QUEUE_SIZE
-#define GNRC_RPL_MSG_QUEUE_SIZE (8U)
+#ifndef CONFIG_GNRC_RPL_MSG_QUEUE_SIZE
+#define CONFIG_GNRC_RPL_MSG_QUEUE_SIZE (8U)
```

Comment:
In #14071 the exponent was used to model the power of 2 restriction of message queues. Should we do this here as well or rather as a follow-up (including the other queue sizes)

Reply:
Good point. Let's defer that to a follow-up PR, though. The diff and commit history is already kind of convoluted.